include/tvm/runtime/disco/disco_worker.h

```diff
@@ -68,6 +69,10 @@ class DiscoWorker {
   int worker_id;
   /*! \brief Total number of workers */
   int num_workers;
+  /*! \brief The local worker id. This can be different from `worker_id` if the session is
+   * consisted of multiple distritributed sub-sessions.
+   */
+  int local_worker_id;
```

Comment:
We introduced `int num_groups` in #17180 which causes some conflicts. `local_worker_id` now equals to `worker_id % (num_workers / num_groups)`.